Dockerfile

```diff
@@ -5,6 +5,8 @@ USER root
 COPY . /app
 WORKDIR /app
 
+RUN apt-get update && apt-get install -y git
```

Comment:
I realize you are debugging but we'll want to add commands to remove data apt caches etc. after this. I forget the precise invocation, but it is common enough to find in various places.

Reply:
But this is just in the build container. I think it'll all be gone after?

Reply:
Ohhhhhh yes, you are correct! Sorry for the noise!